framework/PostProcessors/ETstructure.py

```diff
@@ -144,7 +144,7 @@ def checkETstructure(self,links,listETs,connectivityMatrix):
         if len(zeroColumns)==0:
             raise IOError('No root ET')
         if len(zeroColumns)==1:
-            rootETID = listETs[zeroColumns]
+            rootETID = listETs[zeroColumns.astype(int)[0]]
```

Comment:
why is this needed?

Reply:
This is complained by the new updated libraries.